docs/decisions/009-tokens-structure.md

```diff
@@ -0,0 +1,49 @@
+# Tokens Structure
+
+Date: 2024-11-19
+
+Status: accepted
+
+## Context
+
+Our Design System uses design tokens.
+
+## Decision
+
+This is what we decided on:
+
+1. We divide tokens into `Global Tokens` and `Theme Tokens`. The `Global Tokens` are non-themed tokens – their
+   values are the same in all themes, for example breakpoints. The `Theme Tokens` have the same structure, but in each theme,
+   their value can differ, for example colors.
+2. Global tokens are divided into `Borders`, `Gradients`, `Other`, `Radii`, `Shadows`, `Spacing` and `Typography`. Each category
+   has its own file and contains tokens that are related to the category. The `Other` category contains tokens that do not fit
+   into any other category.
+3. The `Gradients` and `Shadows` tokens use CSS variables to define the colors. This allows us to theme them.
+4. We use scales for tokens that have multiple values. For example, the `Spacing` category contains `space-0`, `space-100`, `space-200`, etc.
```

Comment:
Also, we use the breakpoint suffix for responsive tokens and objects for responsive tokens like Typography.